engines/bastion/README.md

```diff
@@ -188,13 +188,13 @@ Open the file, and add the following lines (with empty lines above and below for
 To setup a development environment, clone the repository or your fork of the repository. From the git checkout, setup the required development dependencies by running:
 
 ```
-rake bastion:setup
+npm install
```

Comment:
Does only `npm install` work to setup everything?

Reply:
looking at https://github.com/theforeman/foreman-packaging/pull/3559

it looks like it does, except for this step:
    sh "cp -rf \$(cat foreman/bastion-version) engines/bastion_katello/bastion-${bastion_version}"
@ehelms would it make sense to just create a symlink there so we don't have to do that step?

Reply:
I only ask because [it looks like more steps](https://github.com/Katello/bastion/blob/master/Rakefile#L35-L43), but not sure if they are redundant. I can test it out soon too

Reply:
Oh and one more step here:  "npm install bastion-${bastion_version}".    Which i'll add to the docs

Reply:
@ehelms could this be replaced with just  'npm install ../bastion' ?

Reply:
i pushed some updated docs, and added simple instructions to ./engines/katello_bastin/README.mc

Reply:
> Does only `npm install` work to setup everything?

I suppose that I forgot we commit all vendor files to source control so there is no need to run bower unless you're updating a dependency.  

It should be possible to remove bower entirely since we aren't really updating dependencies that often and replace it with npm and webpack in a later PR.